src/components/Capacity.svelte

```diff
@@ -37,17 +24,11 @@
 
   let capacityDetails: CapacityDetails = defaultDetails;
 
-  user.subscribe(async (val) => {
-    if ($storeConnected && apiPromise) {
-      blockNumber = await getBlockNumber(apiPromise);
-      $storeBlockNumber = blockNumber;
-    }
-    if ($storeConnected && apiPromise?.query && $user.address) {
-      let info = await getMsaEpochAndCapacityInfo(apiPromise, $user.address);
-
-      capacityDetails = { ...defaultDetails, ...info.capacityDetails };
-      epochNumber = info.epochNumber;
-      storeChainInfo.update((info: ChainInfo) => (info = { ...info, epochNumber }));
+  afterUpdate(async () => {
+    if ($user.address) {
+      let capacityInfo = await getMsaEpochAndCapacityInfo($dotApi.api as ApiPromise, $user.address);
```

Comment:
```suggestion
      let capacityInfo = await getMsaEpochAndCapacityInfo($dotApi.api as ApiPromise, $user.address!);
```

delete the if and just do this.